src/main/webapp/jsp/instructorFeedbackAjaxRemindParticularStudentsModal.jsp

```diff
@@ -2,14 +2,50 @@
 <%@ page import="teammates.common.util.Const"%>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 
-<c:forEach items="${data.responseStatus.noResponse}" var="userToRemindEmail">
-    <div class="checkbox">
-        <label>
-            <input type="checkbox" name="<%= Const.ParamsNames.SUBMISSION_REMIND_USERLIST %>" value="${userToRemindEmail}">
-            ${data.responseStatus.emailNameTable[userToRemindEmail]}
-        </label>
-    </div>
-</c:forEach>
+<table class="table-responsive table table-striped table-bordered">
+    <tr>
+        <th class="student_remind_tick"></th>
+        <th onclick="toggleSort(this);" class="button_sortsection button-sort-none">
+            Section
+            <span class="icon-sort unsorted"></span>
```

Comment:
Data from back-end comes unsorted, so I haven't specified default sorting order. Should I return sorted data from back-end and if yes, then what column should be used for default sorting. Or it may be left as it is ?